Library/Formula/plan9port.rb

```diff
@@ -17,15 +17,10 @@ class Plan9port < Formula
   def install
     ENV["PLAN9_TARGET"] = libexec
 
-    if build.with? "x11"
-      # Make OS X system fonts available to Plan 9
-      File.open("LOCAL.config", "a") do |f|
-        f.puts "FONTSRV=fontsrv"
-      end
-    end
+    # Make OS X system fonts available to Plan 9
+    (buildpath/"LOCAL.config").write "FONTSRV=fontsrv" if build.with? "x11"
```

Comment:
Nope, this file doesn't exist until we create it. Discovered that last night when changing it to `write_append` encountered my raise because the file didn't exist :smile:.